app/controllers/forecasting/distributions_controller.rb

```diff
@@ -0,0 +1,36 @@
+class Forecasting::DistributionsController < ApplicationController
+  def index
+    @series     = series
+    @categories = categories
+  end
+
+  private
+
+  def series
+    items = []
+
+    Item.all.sort.each do |item|
```

Comment:
Unless this is done at the `SuperAdmin` level, this will need to be scoped to the `current_organization`. Something like:
```suggestion
    current_organization.items.sort.each do |item|
```